src/lib/y2partitioner/device_graphs.rb

```diff
@@ -0,0 +1,47 @@
+require "singleton"
+
+module Y2Partitioner
+  # Object that holds and manipulates of storage device graphs.
+  class DeviceGraphs
```

Comment:
@mvidner in fact no, I think we agreed that we can pass object `partition`, just after any modification of graph, we have to redraw dialog, as some of the object can be invalid or no longer exists. If we pass only device_graph and name, that API will look very horrible.
So basically any success modification should result in `:redraw`